ci-operator/step-registry/edge/ci/ocm-publish/edge-ci-ocm-publish-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+echo "Assisted Installer Image Info:\n"
+echo "IMAGE_NAME:     ${IMAGE_NAME}\n"
+echo "IMAGE_TAG:      ${IMAGE_TAG}\n"
+echo "IMAGE_VERSION:  ${IMAGE_VERSION}\n"
+echo "IMAGE_REGISTRY: ${IMAGE_REGISTRY}\n"
+echo "IMAGE_ORG:      ${IMAGE_ORG}\n"
+echo "------------------------------------------------------\n"
+echo "OCM Pipeline Info:\n"
```

Comment:
All these `echo` that use \n` should be `printf` instead. Echo does not process backslash \n